test/libsolidity/util/BytesUtils.cpp

```diff
@@ -81,7 +81,7 @@ bytes BytesUtils::convertBoolean(string const& _literal)
 	else if (_literal == "false")
 		return bytes{false};
 	else
-		throw Error(Error::Type::ParserError, "Boolean literal invalid.");
+		throw Error(6991_error, Error::Type::ParserError, "Boolean literal invalid.");
```

Comment:
Do we want unique IDs in tests?

Reply:
I would say tests should not have IDs at all. Also, this is an exception and not an error that is added to the error list, isn't it?

Reply:
Yes. I replaced it with another, test-specific exception (`TestParserError`, be83c54).